Formula/vifm.rb

```diff
@@ -1,8 +1,9 @@
 class Vifm < Formula
   desc "Ncurses based file manager with vi like keybindings"
   homepage "https://vifm.info/"
-  url "https://github.com/vifm/vifm/releases/download/v0.10.1/vifm-0.10.1.tar.bz2"
-  sha256 "99006f56db05e1bdfb7983e8d5c31c9ac93bf2ac9e0409a577c8ca660fecd03b"
+  url "https://github.com/vifm/vifm/releases/download/v0.11/vifm-osx-0.11.tar.bz2"
+  sha256 "d6f829ed0228a8f534d63479fc988c2b4b95e6bc49d5db5e4fabff337fba3c4c"
```

Comment:
```suggestion
  url "https://github.com/vifm/vifm/releases/download/v0.11/vifm-0.11.tar.bz2"
  sha256 "d06f7e25f1ba0936f27f3d34401cc9eaa50ccc5a5e70ad5659e5014f04691e67"
```

`vifm-osx-0.11.tar.bz2` is the macOS binary. We should be using the source archive, which is `vifm-0.11.tar.bz2`.